GitExtUtils/Delimiters.cs

```diff
@@ -18,6 +18,8 @@ internal static class Delimiters
         public static readonly char[] TabAndLineFeedAndCarriageReturn = { '\t', '\n', '\r' };
         public static readonly char[] NullAndLineFeed = { '\0', '\n' };
         public static readonly char[] LineFeedAndCarriageReturn = { '\n', '\r' };
+        public static readonly char[] LineFeedCarriageReturnAndNull = { '\n', '\r', '\0' };
```

Comment:
❔
```suggestion
        public static readonly char[] LineFeedCarriageReturnAndZero = { '\n', '\r', '\0' };
```

Reply:
I used the same naming than 2 lines above and I think we talk more about Null character https://en.wikipedia.org/wiki/Null_character so I prefer to use that.